### DISCUSSION TITLE
Course page locked at Lesson 1 (13 of 25)

### DISCUSSION THREAD

**Discussion by ytlee93** [OpenMined/courses#315] [Course: Foundations of Private Computation] [answered]
When working on the 'Foundations of Private Computation' course Lesson 1 (13 of 25), the 'next' button is locked and unable to proceed to the next part of the lecture. (There are no quiz questions to answer in that section)
Are there any solutions to continue the course?
![image](https://user-images.githubusercontent.com/80210231/114328765-1f609d80-9b70-11eb-8cd8-3e58edadbfe7.png)
![image](https://user-images.githubusercontent.com/80210231/114328770-238cbb00-9b70-11eb-83e8-68fdbaaf6f4c.png)

**Comment by nigoshh** [comment, marked as answer]
See discussion https://github.com/OpenMined/courses/discussions/272 for a quick fix (decrease the zoom level)

**Reply by ytlee93** [reply]
Thanks a lot! But the course page of mine went normal all of a sudden without adjusting the zoom level.
